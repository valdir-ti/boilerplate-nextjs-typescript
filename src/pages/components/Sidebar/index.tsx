import { Container } from '../../../styles/sidebar'

const Sidebar = () => {
  return (
    <Container>
      <h2>Sidebar</h2>
    </Container>
  )
}

export default Sidebar
