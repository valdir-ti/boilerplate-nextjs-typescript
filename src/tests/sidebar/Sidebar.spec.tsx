import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'

import Sidebar from '../../pages/components/Sidebar'

describe('<Sidebar />', () => {
  it('should render a heading', () => {
    render(<Sidebar />)
    expect(
      screen.getByRole('heading', { name: /Sidebar/i })
    ).toBeInTheDocument()
  })

  it('should match a snapshot', () => {
    const { container } = render(<Sidebar />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
