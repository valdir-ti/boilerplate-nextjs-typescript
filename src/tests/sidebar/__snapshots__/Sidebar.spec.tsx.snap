// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sidebar /> should match a snapshot 1`] = `
<div
  class="sc-bcXHqe cJDoow"
>
  <h2>
    Sidebar
  </h2>
</div>
`;
